### ISSUE TITLE
Notifications with sub-application/deployment name

### ISSUE THREAD

**Issue by AshwiniSwain** [argoproj-labs/argocd-notifications#259]
Hi guys

Currently I am using ArgoCD Notifications with slack, and get notifications every time a sync is successful/un-successful. The sync is triggered using the argocd cli through a jenkins job.

Many of my applications have multiple deployments(for different micro-services), and I don't sync the entire application all at once, rather just a single deployment in that application.

Is there a way to get the name of the deployment/sub-application in the notifications?? 

Currently I use : `.app.metadata.name` for the application name, but was not able to find out a way to get the specific subset that was synced.

Is it possible with ArgoCD notifications??

Thanks

**Comment by 912988434** [comment]
Yes, sometimes when I update a deployment in the app, what I want is the synchronization notification of this depolyment, but now I get the notification of the whole app synchronization

**Comment by AshwiniSwain** [comment]
@912988434 I was able to get it working using `{{ (index .app.status.operationState.operation.sync.resources 0).name }}`.

**Comment by 912988434** [comment]
> {{ (index .app.status.operationState.operation.sync.resources 0).name }}

This will get the names of all the synchronized subroutines, but I prefer to show which subroutines were synchronized this time in the notification, which seems to be difficult to implement.
